pkg/paymasterclient/client.go

```diff
@@ -0,0 +1,104 @@
+package paymasterclient
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/gofrs/uuid"
+)
+
+type Client interface {
+	// ChainID returns the chain ID of the connected domain
+	ChainID(ctx context.Context) (string, error)
+	// IsSponsorable checks if a transaction is sponsorable
```

Comment:
some missing APIs , like `TransactioNum`, please check the latest code: 1. update the interface; 2. update the data structure

Reply:
added